migration/v0.34/README.md

```diff
@@ -113,6 +129,13 @@ As with all scripts in this repository `CK8S_CONFIG_PATH` is expected to be set.
 
 > *Done during maintenance window.*
 
+1. Delete obsolete namespaces:
+
+    ```bash
+    ./bin/ck8s ops kubectl sc delete ns elastic-system
+    ./bin/ck8s ops kubectl sc delete ns influxdb-prometheus
```

Comment:
same as above comment but also move it to the right order to around [here](https://github.com/elastisys/compliantkubernetes-apps/blob/d0d207a532078d64d084b1398b7b0401cad657d5/migration/v0.34/README.md?plain=1#L152)

Reply:
Like 68655f32348181ebd71e10f087cdecb496239297 then

Reply:
exactly